plugins/compliance-hub/frontend/public/templates/user.html

```diff
@@ -38,6 +38,34 @@
 
                     </template>
                 </el-table-column>
+                <el-table-column>
+                    <template v-slot="rowScope">
+                        <cly-more-options  size="small">
```

Comment:
Since we need to show the action button on hover.
You need to a if condition here - 
`v-if="rowScope.row.hover"`